config/software/gcc.rb

```diff
@@ -19,7 +20,7 @@
 # It would be nice to use the real GCC version number here, but the
 # Omnibus Git fetcher uses this value as the ref name to fetch and
 # there appears to be no way to override that.
-default_version "codethink/fortran-extra-legacy-7.2"
+default_version "#{ENV['OMNIBUS_GCC_GIT_REF']}"
```

Comment:
I wonder about naming this variable 'OMNIBUS_GIT_REF' instead. In the future, we may need to pass LLVM, Clang or Flang git ref.

Reply:
I prefer to keep GCC in the name for the same reason you wrote in this comment, the reference for these packages may or may not be the same for all of them (normally it won't be), so I will keep this variable name as it is.